### PR TITLE
Populate newsfeed

### LINKED ISSUE #50: Populate Newsfeed
Pull DB questions/answers into newsfeed and replace hard-coded data to dynamically display posts.

### PR DESCRIPTION
**Pull request by Code-zart**
closes #50

### CODE REVIEW

File: client/src/components/Post/Post.jsx [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
-import NearMeIcon from '@material-ui/icons/NearMe';
-import AccountCircleIcon from '@material-ui/icons/AccountCircle';
-import { ExpandMoreOutlined } from '@material-ui/icons';
+import { AppContext } from '../../context/AppContext';
+
+const Post = ({ questions }) => {
+  // pass answers into "answer" component
+  const { answers } = useContext(AppContext);
+
+  // posts.map((post) => {
```
Comment (ErnieAtLYD) [inline client/src/components/Post/Post.jsx:14]: Just make sure you remember to remove the comments before final code freeze and we're all good.

File: client/src/components/Post/Post.jsx [hunk 2]
```diff
@@ -1,55 +1,57 @@
...
-          <NearMeIcon />
-          <p>Share</p>
-        </div>
-        <div className="post__option">
-          <AccountCircleIcon />
-          <ExpandMoreOutlined />
+          <div className="post__options">
+            <div id="upvote" className="post__option">
+              <ThumbUpIcon />
+              <p>{question.upvotes.length}</p>
```
Comment (ErnieAtLYD) [inline client/src/components/Post/Post.jsx:40]: Are we confident that `question.upvotes` and `question.downvotes` will never return `undefined`? If not then I would go `{question?.upvotes?.length}` assuming you're running this on the latest node version build.

**Review by ErnieAtLYD — APPROVED** [review]
At least at a cursory glance this looks fine but check my comments. -E

File: client/src/components/Post/Post.jsx (hunk 2 above)
Comment (Code-zart) [reply client/src/components/Post/Post.jsx:40]: good call. It's possible, but unlikely, to have zero upvotes so it's a quick change.

**Pull request merged**